### PR TITLE
Heat capacity

### PR DESCRIPTION
**Pull request by gusah153**
Implemented heat capacity and added .traj files which can be accessed in the unit tests. The command_line_parseing broke after a merge of develop into this branch but everything concerning heat-capacity works.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -72,10 +71,11 @@ def LJ(use_asap=use_asap):
 
     # Set the momenta corresponding to T=300K
     MaxwellBoltzmannDistribution(atoms, temperature_K=options["temperature_K"])
+    # Is this where the temperature is halfed??
     Stationary(atoms)
     ZeroRotation(atoms)
     # We want to run MD with constant energy using the VelocityVerlet algorithm.
-    dyn = VelocityVerlet(atoms, 5 * units.fs)  # 5 fs time step.
+    dyn = VelocityVerlet(atoms, 1 * units.fs)  # 5 fs time step.
```
Comment (martinclason) [inline main.py:78]: This should be read from config file, called dt

File: specificHeatCapacity.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+from ase import atoms, units
+import sys
+
+def specificHeatCapacity(ensamble, traj):
+  """This function calculates the specific heat capacity from the trajectory output
+  file and the ensamble that has been simulated."""
+
+  # Calculates the number of atoms and the temperature of the simulated system.
+  N = len(traj[1].get_positions()) # Number of atoms is the same as number of positions
+  T = np.sum([atoms.get_temperature() for atoms in traj[-20:]])/20 # Average of last 10 itterations
```
Comment (martinclason) [inline specificHeatCapacity.py:12]: last 20 instead of 10?

File: main.py (hunk 1 above)
Comment (gusah153) [reply main.py:78]: I haven't touched on these and I don't think the merge from develop covered that, however I  can go and change it.

File: specificHeatCapacity.py (hunk 2 above)
Comment (gusah153) [reply specificHeatCapacity.py:12]: Yes 20.

**Review by gusah153 — COMMENTED** [review]
The timestep has been changed to use dt from the config file and the heat capacity averages the last 20 saved itterations in the simulation.

**Review by gusah153 — COMMENTED** [review]
Restored other ale and createAtoms tests to a working state.

**Review by martinclason — APPROVED** [review]

**Pull request merged**